pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixServerStarter.java

```diff
@@ -103,9 +107,13 @@
 public class HelixServerStarter implements ServiceStartable {
   private static final Logger LOGGER = LoggerFactory.getLogger(HelixServerStarter.class);
 
+  private static final String ENVIRONMENT_IDENTIFIER = "environment";
+  private static final String FAILURE_DOMAIN_IDENTIFIER = "failureDomain";
```

Comment:
these can be in common constants ? given this config can be populated by other components as well ?

Reply:
Done.